File: programming problems/HackerRank/30 days of code/day 6, lets review/day_6_lets_review.java
import java.io.*;
import java.util.*;

public class Solution {
    public static void main(String[] args) {
        
        var scanner = new Scanner(System.in);

        for (var word = scanner.nextLine(); scanner.hasNextLine(); ) {

            word = scanner.nextLine();
            var word1 = new StringBuffer();
            var word2 = new StringBuffer();

            boolean flag = true;
            for (var ch : word.toCharArray()) {
                if (flag)
                    word1.append(ch);
                else
                    word2.append(ch);
                flag = !flag;
            }
            System.out.println(word1.toString() + " " + word2.toString());
        }
    }
}

File: programming problems/HackerRank/30 days of code/day 6, lets review/day_6_lets_review.cs
using System;

class Solution {
    static void Main(String[] args) {

        for (string word = Console.ReadLine(); (word = Console.ReadLine()) != null; ) {
            
            var word1 = new System.Text.StringBuilder("");
            var word2 = new System.Text.StringBuilder("");
            var flag = true;

            foreach (var ch in word) {
                if (flag)
                    word1.Append(ch);
                else
                    word2.Append(ch);
                flag = !flag;
            }
            Console.WriteLine($"{word1} {word2}");
        }
    }
}
